server/src/services/mail/index.js

```diff
@@ -0,0 +1,32 @@
+import nodemailer from "nodemailer";
+import { config } from "dotenv";
+config();
+
+const sendEmail = () =>{
```

Comment:
```suggestion
const sendEmail = (data) =>{
```

Please provide data to this function from outside, so when we just want to use it we can pass a data to it